packages/lisk-p2p/src/peer.ts

```diff
@@ -13,45 +13,63 @@
  *
  */
 /* tslint:disable:interface-name */
+import { PeerState } from './p2p_types';
+
+// TODO: Use to create outbound socket connection inside peer object.
+// TODO: const socketClusterClient = require('socketcluster-client');
 
 export interface IPeerConfig {
 	readonly clock?: Date;
 	readonly height: number;
-	readonly httpPort?: number;
-	readonly ip: string;
+	readonly id: string;
+	readonly inboundSocket?: any; // TODO: Type SCServerSocket
+	readonly ipAddress: string;
 	readonly os?: string;
-	readonly state?: number;
 	readonly version?: string;
 	readonly wsPort: number;
 }
 
 export class Peer {
-	private height: number;
-	private ip: string;
-	private wsPort: number;
+	private _height: number;
+	private readonly _id: string;
+	private readonly _inboundSocket: any;
+	private readonly _ipAddress: string;
+	private readonly _wsPort: number;
 
 	public constructor(peerConfig: IPeerConfig) {
-		this.height = peerConfig.height;
-		this.ip = peerConfig.ip;
-		this.wsPort = peerConfig.wsPort;
+		this._id = peerConfig.id;
+		this._ipAddress = peerConfig.ipAddress;
+		this._wsPort = peerConfig.wsPort;
+		this._inboundSocket = peerConfig.inboundSocket;
+		this._height = peerConfig.height;
 	}
 
 	public getHeight(): number {
-		return this.height;
+		return this._height;
+	}
+
+	public getId(): string {
```

Comment:
i'm not sure if this is scope of  Ishan's PR or this PR, but it should be 
`public get id(): string {`
@ishantiw

Reply:
I think we can ignore it here as it will be covered by my PR